packages/admin/docs/01-installation.md

```diff
@@ -61,6 +61,7 @@ If you wish to translate the package, you may publish the language files using:
 
 ```bash
 php artisan vendor:publish --tag=filament-translations
+php artisan vendor:publish --tag=filament-support-translations
```

Comment:
Yep, `--tag=foo --tag=bar`.

Reply:
Uhm, I just wanted to push up the complete command, but it seems I'm unable to push to this PR.

Reply:
there is, but it can confuse people into thinking they need to run the command with all the tags.
I'd keep it 1 command per tag.

Regarding @zepfietje "include forms and tables as well", i agree, it is best to list all possible/available tags